core/modules/qdisp/MessageStore.cc

```diff
@@ -39,11 +39,44 @@ namespace lsst {
 namespace qserv {
 namespace qdisp {
 
+namespace {
+	const int NOTSET = -1;
+}
+
 ////////////////////////////////////////////////////////////////////////
 // public
 ////////////////////////////////////////////////////////////////////////
 
-void MessageStore::addMessage(int chunkId, int code, std::string const& description) {
+/** Output operator for Severity */
+std::ostream& operator<<(std::ostream& os, Severity const& severity) {
+	switch (severity) {
+	case INFO:
+		os << "INFO";
+		break;
+	case ERROR:
+		os << "ERROR";
+		break;
+	default:
+		os << "UNKNOWN";
+	}
+	return os;
+}
+
+/** Convert Severity to std::string */
+std::string to_string(Severity const& severity) {
```

Comment:
severity enum is now swigged, as a consequence this method was unused and then removed.